raiden_contracts/data/deployment_goerli.json

```diff
@@ -1 +1 @@
-{"contracts_version": null, "chain_id": 5, "contracts": {"SecretRegistry": {"address": "0xF6d610f7c8473a0F0c8c1653AF28A68C69860288", "transaction_hash": "0x9e4d32a73673f046e731580c2a6aaafc28a9ca0aaecee689fb61e28754abcb82", "block_number": 875035, "gas_cost": 298578, "constructor_arguments": []}, "TokenNetworkRegistry": {"address": "0x32eC010f491620e0854Dc8E3243186AC55453449", "transaction_hash": "0x9b939148f1a8b55f498d398769fde50684dc2b5b7860f0e9b8e666e41b175886", "block_number": 991445, "gas_cost": 5029873, "constructor_arguments": ["0xF6d610f7c8473a0F0c8c1653AF28A68C69860288", 5, 500, 555428, 115792089237316195423570985008687907853269984665640564039457584007913129639935]}}}
\ No newline at end of file
+{"contracts_version": null, "chain_id": 5, "contracts": {"SecretRegistry": {"address": "0xF6d610f7c8473a0F0c8c1653AF28A68C69860288", "transaction_hash": "0x9e4d32a73673f046e731580c2a6aaafc28a9ca0aaecee689fb61e28754abcb82", "block_number": 875035, "gas_cost": 298578, "constructor_arguments": []}, "TokenNetworkRegistry": {"address": "0xd943A9BA405B95f22517E4AC3E883cf8749B84D3", "transaction_hash": "0x8b98303f019dc5ca4f8a1c80a140ed2c13592d3939f13909a7ce5dd7c3d68b2d", "block_number": 1007698, "gas_cost": 5029873, "constructor_arguments": ["0xF6d610f7c8473a0F0c8c1653AF28A68C69860288", 5, 500, 555428, 115792089237316195423570985008687907853269984665640564039457584007913129639935]}}}
```

Comment:
Contracts were deployed on Goerli for this release.  The SecretRegistry instance is being reused.